code/integration-tests/runtime-tests/tests/utils/constants.ts

```diff
@@ -0,0 +1,22 @@
+export const picassoEndpoint = 'ws://localhost:9988';
+export const composableEndpoint = 'ws://localhost:29988';
+export const centauriEndpoint = 'http://127.0.0.1:26657';
+export const osmosisEndpoint = 'http://127.0.0.1:36657';
+export const kusamaEndpoint = 'ws://localhost:9944';
+export const  polkadotEndpoint = 'ws://localhost:29944';
+export const picaIdOnComposable = '79228162514264337593543950337';
```

Comment:
is not cppaste of assets.json?

Reply:
Correct, removed the constants file. And moved all the logic into config.json file.